packages/nx/src/project-graph/build-project-graph.ts

```diff
@@ -177,11 +181,28 @@ function jsPluginConfig(nxJson: NxJsonConfiguration): NrwlJsPluginConfig {
   return nxJson?.pluginsConfig?.['@nrwl/js'] ?? {};
 }
 
+interface NxCorePluginConfiguration {
+  includeImplicitExecutorDependencies?: boolean;
```

Comment:
These are not implicit anymore. They are explicit since there's code that we can statically analyze to find them.

I wouldn't mind if this was actually on all the time. Also, not just for local plugins, but for npm projects as well.

Reply:
They **_can't_** be on all the time because it can cause circular dependency issues, using the Nx repo as an example.

@nrwl/jest uses @nrwl/js:tsc to build, so @nrwl/jest -> @nrwl/js
@nrwl/js uses @nrwl/jest:jest to test, so @nrwl/js -> @nrwl/jest

If these are enabled, `nx build jest` fails with a circular dep error.

Reply:
FWIW though, the npm project detection is in this PR.